src/kivy_garden/qrcode/qrcode_widget.py

```diff
@@ -29,6 +29,13 @@ class QRCodeWidget(FloatLayout):
     `None`.
     '''
 
+    error_correction = NumericProperty(qrcode.constants.ERROR_CORRECT_L)
+    ''' The error correction level for the qrcode.
+
+    :data:`error_correction` is a constant in :module:`~qrcode.constants`, defaulting
+    to `qrcode.constants.ERROR_CORRECT_L`.
+    '''
```

Comment:
Minor: docstring should be using double quotes.
https://www.python.org/dev/peps/pep-0257/#what-is-a-docstring
I'll address after merging, since I need to fix the CI anyway